Code/Python/calculate_the_value_of_nCr.py

```diff
@@ -0,0 +1,65 @@
+#This is the program to calcuate the nCr.
+
+#In this method used for finding nCr ,while calculating nCr   n(n-1)(n-2)---(n-r+1)/r(r-1)---1
+#we divide both numerator and denominator by gcd() numbers as to make calculation simple 
+#and to ensue the result may not go out of range
+
+
+def gcd(num1,num2):             #This function is for calculating greatest common divisor
```

Comment:
dont give this much space between comment and def, rather use docstring for the function i.e.

```python
def func():
    """ XYZ.... """
```

do this for all the functions in this file